packages/jest-test-result/src/types.ts

```diff
@@ -104,6 +104,7 @@ export type TestResult = {
     unmatched: number;
     updated: number;
   };
+  // TODO - Remove in Jest 26
   sourceMaps?: {
```

Comment:
are there other use cases than coverage for having access to sourcemaps in results? If not, removing it will be great, will probably free up a bunch of memory

Reply:
Only use I saw was for registering the source maps with Istanbul's `MapStore` so that they would be used for remapping coverage through `MapStore.transformCoverage`.